### ISSUE TITLE
Fresh install - no OpenOutreach Platform available?

### ISSUE THREAD

**Issue by civiclix** [omega8cc/boa#637]
Totally fresh install. I'd like to install OpenOutreach, but it is not available under my platforms...? I thought I had selected it in the initial install, but didn't see it, so ran the upgrade again, hoping I had missed it, but not there at all, either. ;-(

Looked at the distribution on drupal.org, but confused as to how to do it the "BOA Way". Any pointers/doc? I was able to create a simple CiviCRM site by creating a makefile in ~/static, running drush make, and verifying the server.

Been a while since I've used Aegir in Drush, so please forgive my stupidity!

Thanks...I'm LOVING BOA compared to "Vanilla" Aegir...really will make my life much easier once I get a leg up on the learning curve!

**Comment by civiclix** [comment]
Saw there was an issue...https://www.drupal.org/node/2293949.

Safe to try to install in /static?

Thanks!

**Comment by bserem** [comment]
Define 'safe' ?
It will work just like all other static platforms. Keep in mind that boa also has a "platformles flavor". So yeah, you go on and try it. Just make sure to get the same make files as boa does
On Mar 2, 2015 6:08 AM, civiclix <notifications@github.com> wrote:Saw there was an issue...https://www.drupal.org/node/2293949.

Safe to try to install in /static?

Thanks!

**Comment by civiclix** [comment]
I guess that's my question...I saw in the platfoms doc on github that there should be a 'canned' platform available to install, but wasn't prompted to install it ('ask mode'). I assumed it was taken out because of the issue I found. Is that what you mean by 'platformles[s?] flavor'. What I was planning to do was install from the distribution and switch the core to the BOA core...that's what I did to install CiviCRM.

Thanks for your help, Bill!

---- On Mon, 02 Mar 2015 06:10:29 +0000 Bill Seremetis &lt;notifications@github.com&gt; wrote ---- 

Define 'safe' ?
 It will work just like all other static platforms. Keep in mind that boa also has a "platformles flavor". So yeah, you go on and try it. Just make sure to get the same make files as boa does
 On Mar 2, 2015 6:08 AM, civiclix &lt;notifications@github.com&gt; wrote:Saw there was an issue...https://www.drupal.org/node/2293949. 
 
 Safe to try to install in /static? 
 
 Thanks!

**Comment by omega8cc** [comment]
OpenOutreach is included by default and can be also added on upgrade, if needed.

**Comment by omega8cc** [comment]
For reference:

https://github.com/omega8cc/boa/blob/master/docs/cnf/octopus.cnf#L12
https://github.com/omega8cc/boa/blob/master/docs/PLATFORMS.txt
https://github.com/omega8cc/boa/blob/master/aegir/tools/system/conf/control-readme.txt#L165

**Comment by civiclix** [comment]
Thanks for the response...unfortunately, something's wrong, then...have done a fresh install, and openoutreach is NOT in the list of platforms (I answered y to all prompts). I am confused, because it and CiviCRM are in the platforms.txt file, and also in the octopus file, so I assumed they would be installed, but they're not.

Am I doing something wrong? Forgive my ignorance...I used to use Aegir on my own, but it's been a few years, and I'm trying to wrap my head around getting things done in BOA. Back then, to create my platforms, I'd just add - say - D7 to my platforms directory, create a profile for the special things I'd want to use (say, CiviCRM), verify everything, and install a site.

Tried to follow that methodology with OpenOutreach this morning with no joy...didn't work. Been reading thru all the doc, so SLOWLY getting used to this...

Thanks for your help and the great work on this! Just a matter of me learning the BOA way.
